src/lesson_perception/CMakeLists.txt

```diff
@@ -0,0 +1,180 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(lesson_perception)
+add_definitions(-std=c++11)
+## Find catkin macros and libraries
```

Comment:
Third step in Create a Python Package (https://industrial-training-master.readthedocs.io/en/melodic/_source/session5/Simple-PCL-Interface-for-Python.html) needs to say "Uncomment line 18 instead of 23"